low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -415,6 +415,8 @@ def _generate_host_update(self):
                 # Transmit it out over rabbitMQ channel
                 self.host_sensor_data = jsonMsg
                 self.os_sensor_type["memory_usage"] = self.host_sensor_data
+                # RAAL stands for RAise ALert
+                logger.info(f"RAAL: {jsonMsg}")

```

Comment:
here even if its logged, doesnt mean its delivered to rabbitmq for sure, which we need

Reply:
Agreed.